README.md

```diff
@@ -20,7 +20,7 @@ Note: This project is currently a work-in-progress, so expect the hardware and s
 ## Project Layout
 
 ### Hardware
-The "hardware" directory contains [KiCad](http://kicad-pcb.org/) projects
+The "hardware" directory contains [KiCad](http://kicad.org/) projects
```

Comment:
Please change the URL to: https://www.kicad.org/
This way it will match what the site actually redirects you to.

Reply:
done